src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/chapterPage/viewModel/ChapterPageViewModel.kt

```diff
@@ -0,0 +1,58 @@
+package org.wycliffeassociates.otter.jvm.app.ui.chapterPage.viewModel
+
+import io.reactivex.rxkotlin.subscribeBy
+import javafx.collections.FXCollections
+import org.wycliffeassociates.otter.jvm.app.ui.chapterPage.model.ChapterPageModel
+import org.wycliffeassociates.otter.jvm.app.ui.chapterPage.model.Verse
+import org.wycliffeassociates.otter.jvm.app.ui.inject.Injector
+import tornadofx.*
+
+class ChapterPageViewModel : ViewModel() {
+    val model = ChapterPageModel()
+//    val chapterPageUseCase = ChapterPageUseCase(Injector.chapterDao)
+    private val modelChapters = model.chapters
+    val activeChapter = model.activeChapter
+    val selectedTab = model.selectedTab
+    val chapters = FXCollections.observableArrayList<String>()!!
+    val verses = FXCollections.observableArrayList<Verse>()!!
```

Comment:
Can we remove the `!!`s? Could be wrong, but I don't think `FXCollections.observableArrayList<>()` returns a nullable

Reply:
no it doesn't return a nullable, but IntelliJ was just complaining about it. I believe it's because it's from a Java Library and kotlin just doesn't know how to make any assumptions about it.